src/AutoCrudAdmin/Constants.cs

```diff
@@ -109,5 +109,10 @@ public static class PartialView
         /// Partial view name for the entity form control.
         /// </summary>
         public const string EntityFormControlPartial = "_EntityFormControlPartial";
+
+        /// <summary>
+        /// Partial view name for the entity form controls.
+        /// </summary>
+        public const string EntityFormControlsPartial = "_EnityFormControlsPartial";
```

Comment:
Why are there two of those?

Reply:
The first one is the partial view that accepts a single form control view model and the second one works with multiple form control view models, which is what we need for the expandable multichoice checkbox.